bigml/batchpredictionhandler.py

```diff
@@ -46,7 +46,8 @@ def __init__(self):
            instantiated independently.
 
         """
-        self.batch_prediction_url = self.url + BATCH_PREDICTION_PATH
+        self.batch_prediction_url = (self.prediction_base_url +
```

Comment:
I don't think the predict server does batch predictions... does it? @sdesimone ?